src/pages/trade-types/components/_timeline.tsx

```diff
@@ -34,7 +36,7 @@ const Timeline = ({ children, ...props }) => {
                     <Oval src={Checklist} alt="checklist" />
                     <Container>
                         <Header mb="0.8rem" mt="-5px" as="h4" size="2.4rem">
-                            {child.props.title}
+                            <Localize translate_text={child.props.title as TString} />
```

Comment:
We trying it not to use `as`, what kind of error shows here?

Reply:
Since title has the type of any, i use as to cast it to TString because translate_text only accept TString